policy/release/sbom_cyclonedx/sbom_cyclonedx.rego

```diff
@@ -148,6 +148,63 @@ deny contains result if {
 	result := lib.result_helper_with_term(rego.metadata.chain(), [id, reference.url, reference.type, msg], id)
 }
 
+# METADATA
+# title: Allowed package sources
+# description: >-
+#   Confirm the CycloneDX SBOM packages fetched by Cachi2 that define externalReferences
```

Comment:
This is a bit hard to read. Let's try to make it nicer since it's user-facing, consider:

> For each of the components fetched by Cachi2 which define `externalReferences` of type `distribution`, verify they are allowed based on the `allowed_package_sources` rule data key. By default, `allowed_package_sources` is empty which means no components with such references are allowed.

(We should call them components, not packages, as per the CycloneDX spec.)

Reply:
Fixed.